DuckDuckGo/KeychainItemsDebugViewController.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  KeychainItemsDebugViewController.swift
+//  DuckDuckGo
+//
+//  Copyright © 2021 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import UIKit
+import WebKit
+import Core
+
+private struct KeychainItem {
+    
+    let secClass: SecClass
+    let service: String?
+    let account: String?
+    let valueData: Data?
+    
+    var value: String? {
+        guard let valueData = valueData else { return nil }
+        return String(data: valueData, encoding: .utf8)
+    }
+    
+    // swiftlint:disable line_length
+    var displayString: String {
+        return "Service: \(service ?? "nil")\nAccount: \(account ?? "nil")\nValue as String: \(value ?? "nil")\nValue data: \(String(describing: valueData))"
```

Comment:
One small suggestion, you could do something like:
```
    var displayString: String {
        return """
            Service: \(service ?? "nil")
            Account: \(account ?? "nil")
            Value as String: \(value ?? "nil")
            Value data: \(String(describing: valueData))
            """
    }
    
```
    and remove the lint disable rule

Reply:
Definitely better ;)